internal/pkg/util/yamlprotoconverter.go

```diff
@@ -0,0 +1,19 @@
+package yamlprotoconverter
```

Comment:
nit: how about `yamlproto.go`? in the spirit of `google.golang.org/protobuf/encoding/protojson`

Reply:
good idea!